storage/indexer/src/db_indexer.rs

```diff
@@ -395,11 +403,15 @@ impl DBIndexer {
                             .expect("Failed to put state keys to a batch");
                     }
                 });
+                batch.put::<InternalIndexerMetadataSchema>(
```

Comment:
progress should be put into the batch outside of `for_each()`

Reply:
Done